internal/dependencymanager/dependencyinstaller.go

```diff
@@ -275,6 +277,14 @@ func (di *DependencyInstaller) handleFoundContract(networkName, contractAddr, as
 		}
 	}
 
```

Comment:
quick comment here would be good, double negatives can be tricky to read

Reply:
Added a comment